build-release-tools/application/commit_status_setter.py

```diff
@@ -57,7 +58,10 @@ def set_commit_status(repo, pull_id, status, description, build_url = None):
     print "Set commit status for {0} pull {1} successfully".format(repo, pull_id)
 
 def set_commit_status_bat(pr_list, status, build_url = None):
-    description = "Build finished."
+    gmt = time.gmtime()
+    gmtStr = time.strftime("%b %d %a %H:%M:%S %Y", gmt) + ' UTC-12'
```

Comment:
Is this script running in vmslaves? is the time setting there always correct and align with github? Could it be possible to use UTC/GMT instead of hard coded UTC-12 ?